go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -163,8 +149,25 @@ func deleteVIPsFromOVN(vips sets.String, st *serviceTracker, name, namespace, cl
 				}
 			}
 		}
-		// Delete the Service VIP from the Service Tracker
-		st.deleteServiceVIP(name, namespace, vip, proto)
+	}
+	return nil
+}
+
+func deleteVIPsFromIdlingBalancer(vips sets.String, st *serviceTracker, name, namespace string) error {
```

Comment:
is the serviceTracker used inside the function?

Reply:
if not we can avoid injecting the tracker dependency

Reply:
Removed, thanks